mods/persistence/game/objects/items/devices/radio/stack_radio.dm

```diff
@@ -0,0 +1,49 @@
+/obj/item/radio/stack_radio
+	name = "cortical receiver"
+	icon = 'icons/obj/items/device/radio/beacon.dmi'
+	frequency = COR_FREQ
+	canhear_range = 0
+
+	power_usage = 0
+
+// No modification of stack_radios, if they're somehow accessed.
+/obj/item/radio/stack_radio/attack_self()
+	return
+
+// Skipping a lot of checks here since cortical chat is permanently active, for most purposes.
+/obj/item/radio/stack_radio/talk_into(var/mob/living/carbon/M, message, var/verb = "says", var/decl/language/speaking = null)
+	if(!M || !istype(M) || !message) return 0
+
+	var/obj/item/organ/internal/stack/mob_stack = M.internal_organs_by_name[BP_STACK]
+
+	if(!mob_stack) return 0
+
+	if(speaking && (speaking.flags & (NONVERBAL|SIGNLANG))) return 0
+
+	// Sedation chemical effect prevents cortical chat use as well.
+	var/mob/living/carbon/C = M
+	if ((istype(C)) && (C.chem_effects[CE_SEDATE] || C.incapacitated(INCAPACITATION_DISRUPTED)))
+		to_chat(M, SPAN_WARNING("You're unable to reach \the [src]."))
+		return 0
+
+	if((istype(C)) && C.radio_interrupt_cooldown > world.time)
```

Comment:
Can't be interrupted /w cooldowns when you're just projecting thoughts.

Reply:
Not a problem, I can cut this, although I should note that currently it's not just projecting thoughts, because nearby players can hear the whisper. It's not a big deal since it's global anyway, but I'd have to override more files if you'd like me to prevent that.

Reply:
That's probably closer to what we want. If you're sick of this PR you can make it an #issue and do it later.

Reply:
I'll see how complicated it is. Say code is a mess, but it might still be feasible.